src/server/middleware/immutable/error.go

```diff
@@ -1,7 +1,6 @@
 package immutable
 
 import "fmt"
-
```

Comment:
same comment as this one:
* https://github.com/goharbor/harbor/pull/17131/files#r923010380